src/main/frontend/index.html

```diff
@@ -545,6 +545,18 @@
       </article>
     </div>
   </div>
+<div class="tile is-parent">
+  <article class="tile is-child">
+  <label>Display Settings</label>
+      <div class="tabs is-toggle">
```

Comment:
I tried this and I think it looks better as a toggle switch actually, but you are welcome to switch it to the select if you think it is better